tests/cypress/tests/e2e/mlr/form.stepdef.js

```diff
@@ -89,6 +89,7 @@ const completeOverlayForm = (overlayForm) => {
 const processField = (field) => {
   //only try to fill it out if it's enabled
   if (!field.props?.disabled) {
+    cy.wait(600);
```

Comment:
Adding this wait helps make sure the local AWS stuff doesn't implode from too many requests at once